modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js

```diff
@@ -423,6 +446,12 @@ export default class HeatmapLayer extends CompositeLayer {
     });
     this._updateMaxWeightValue();
 
+    // reset filtering parameters
+    weightsTexture.setParameters({
```

Comment:
Why is this needed? Are parameters changed elsewhere?

Reply:
Yes, `Transform` sets parameters to NEAREST to correctly access each individual pixel, we need a proper push pop technique that should happen within `Transform`, tracked by https://github.com/uber/luma.gl/issues/1193